render/process.go

```diff
@@ -22,7 +22,7 @@ func renderProcesses(rpt report.Report) bool {
 }
 
 // EndpointRenderer is a Renderer which produces a renderable endpoint graph.
-var EndpointRenderer = FilterProcspiedOrEBPF(SelectEndpoint)
+var EndpointRenderer = SelectEndpoint
```

Comment:
That's what I had originally, but there is a general theme in the code of having a renderer for every topology. So, for example, in
```golang
var HostRenderer = MakeReduce(
	MakeMap(
		MapEndpoint2Host,
		EndpointRenderer,
	),
	MakeMap(
		MapX2Host,
		ColorConnectedProcessRenderer,
	),
	MakeMap(
		MapX2Host,
		ContainerRenderer,
	),
	MakeMap(
		MapX2Host,
		ContainerImageRenderer,
	),
	MakeMap(
		MapX2Host,
		PodRenderer,
	),
	SelectHost,
)
```
changing `EndpointRenderer` to `SelectEndpoint` does make the latter stick out as different.